lib/ex_cell/adapters/cell_js.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ExCell.Adapters.CellJS do
+  @moduledoc false
```

Comment:
Maybe we should include these in the docs. E.g. a [link to CellJS](https://github.com/DefactoSoftware/cells-js) and how to initialize the adapter?